src/mpv_calendar/templates/index.html

```diff
@@ -217,18 +217,24 @@ <h1>MPV Service Control</h1>
             <button type="submit" class="button">Toggle</button>
         </form>
 
-        <form action="/next">
-            <button type="submit" class="button">Next</button>
-        </form>
         <form action="/previous">
             <button type="submit" class="button">Previous</button>
         </form>
 
+        <form action="/next">
+            <button type="submit" class="button">Next</button>
+        </form>
+
         <form action="/add" class="add post">
-            <input type="text" name="item" placeholder="Add URL or file path">
+            <input type="text" name="item" placeholder="Play URL or file">
             <button type="submit" class="button">Add</button>
         </form>
 
+        <form action="/queue" class="add post">
+            <input type="text" name="item" placeholder="Queue URL or file">
+            <button type="submit" class="button">Queue</button>
+        </form>
+
```

Comment:
UI implementation for Queue vs. Play/Add form field: adds a 2nd form field with a 2nd associated button.